app/src/main/java/com/eveningoutpost/dexdrip/EditAlertActivity.java

```diff
@@ -350,30 +357,52 @@ private boolean verifyThreshold(double threshold) {
         return true;
     }
 
+    private double parseDouble(String str) {
```

Comment:
ahhh I didnt realize you had defined this method, I was wondering how on earth this was suppsoed to work, now I get it